node/service/src/lib.rs

```diff
@@ -56,7 +56,7 @@ pub use {
 	sp_authority_discovery::AuthorityDiscoveryApi,
 	sc_client_api::AuxStore,
 	polkadot_primitives::v1::ParachainHost,
-	polkadot_overseer::{Overseer, OverseerHandler},
+	polkadot_overseer::{Overseer, Handler},
```

Comment:
Why is this called handler and not handle?

Reply:
Because that's what it does. While it is a handle, it does the handling of the overseer, and thus is a handler. But I see your point, renamed.